akka-http2-support/src/main/scala/akka/http/impl/engine/http2/BufferedOutletSupport.scala

```diff
@@ -29,15 +29,17 @@ trait BufferedOutletSupport { logic: GraphStageLogic ⇒
     val buffer: java.util.ArrayDeque[T] = new java.util.ArrayDeque[T]
     var pulled: Boolean = false
 
+    def doPush(elem: T): Unit = outlet.push(elem)
```

Comment:
Ok I see, and I think i meant a different one.
Please add override wherever possible (here), then it's clear by looking at the code that this is the hook, and not a random method we extracted here (I'll do that in my tonights PRs)